### ISSUE TITLE
Comparing to TotalSegmentator models

### ISSUE THREAD

**Issue by FabianIsensee** [wasserth/TotalSegmentator#553]
Hey Jakob,
for some project on intensity normalization I recently ran trainings on the Totalsegmentator V2 dataset. Out of curiosity (and to ground my results) I wanted to compare Dice scores to the official Totalsegmentator models. Unfortunately, as you highlight in [train_nnunet.md](https://github.com/wasserth/TotalSegmentator/blob/master/resources/train_nnunet.md), a comparison with the results of the totalsegmentator models (or your [results file](https://github.com/wasserth/TotalSegmentator/blob/master/resources/evaluate_results.txt)) is flawed because the publicly released dataset does not contain all cases used for training the totalsegmentator models. 
Thus, I was wondering whether you happen to have any results using just the released data (using the official train/val/test split described in `meta.csv`) that you would be willing to share? That would be quite useful to ensure we are not heading in the wrong direction.
Best,
Fabian

**Comment by FabianIsensee** [comment]
[muscles.txt](https://github.com/user-attachments/files/26048663/muscles.txt)
[ribs.txt](https://github.com/user-attachments/files/26048660/ribs.txt)
[cardiac.txt](https://github.com/user-attachments/files/26048661/cardiac.txt)
[vertebrae.txt](https://github.com/user-attachments/files/26048659/vertebrae.txt)
[organs.txt](https://github.com/user-attachments/files/26048662/organs.txt)

Looks like this pretty much reproduces the results reported in your [evaluate.txt](https://github.com/wasserth/TotalSegmentator/blob/master/resources/evaluate_results.txt). Thanks a lot!

**Comment by FabianIsensee** [comment]
Hey @wasserth , awesome work on the v3 version of the dataset. Do you happen to have Dice scores for the test set of that dataset as well? Just to confirm: The 5 v3 models were not trained on the test set so I should be able to reproduce that?

**Comment by wasserth** [comment]
Hi Fabian, totalseg v3 is almost ready. I already published the dataset but found a last minute bug in my model so i have to do a few more fixes before I will merge the totalseg_v3 branch into master. Then I will also have some Dice scores. The 5 v3 models are not trained on the "test" cases (see meta.csv file in the dataset). But only on the "train" cases. There are no "val" cases anymore for v3. We do not use a validation set in v3 anymore only a test set.

**Comment by FabianIsensee** [comment]
Thanks, I thought the new models are out already but maybe I was mistaken :-) Looking forward to the results once it's all in place!
